### ISSUE TITLE
Field "Push items" has a visible character and doesn't give the player any inclination as to what's going on with the field

### ISSUE THREAD

**Issue by CoroNaut** [CleverRaven/Cataclysm-DDA#70235]
### Describe the bug

I'm not entirely sure if this is a bug or not, but the "Push items" field generated with a tear in reality is visible. It looks like its a field that is meant to be invisible?

### Attach save file

[Lenora-trimmed.tar.gz](https://github.com/CleverRaven/Cataclysm-DDA/files/13690067/Lenora-trimmed.tar.gz)

### Steps to reproduce

1. Mapgen, comes with tear in reality

### Expected behavior

I tried putting items in the area where the "push items" field is, but it doesn't really push items, so I don't know what its supposed to do. It just kind of seems like it doesn't do anything, but if it does, it can probably be invisible? If it doesn't or was used for something that's no longer in the game, it can probably be removed.

### Screenshots

![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/24307650/c03591c8-fdf9-47ae-8db5-44cbb18d73e3)
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/24307650/d9d97efa-8a5e-40c9-ac81-ce9d53ebec59)

### Versions and configuration

- OS: Windows
    - OS Version: MINGW/CYGWIN/MSYS2 on unknown Windows version
- Game Version: 7773670 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Slowdown Fungal Growth [no_fungal_growth],
    Stats Through Skills [StatsThroughSkills]
]

**Comment by Procyonae** [comment]
It's effect is defined on L775 of map_field.cpp (https://github.com/CleverRaven/Cataclysm-DDA/blob/04b6eb249b041653afa14cafd7b897f621100ae3/src/map_field.cpp#L775) and it's odd to say the least

**Comment by CoroNaut** [comment]
Looking at the code, I got a better grasp of what it does. I tried stepping back and forth between two tiles with "push items" field, and it randomly throws a rock at my character about 1 every 10 times and does damage. It materializes rocks to throw if there is rubble below, if there isn't rubble, it doesn't have rocks to throw and just doesn't do anything.

Other than that, the name of the field doesn't really match what the field does, and it could still be invisible if that's a better solution here.

If it's intended to be seen, I would suggest it be given a more lore-friendly name. Something like "whooshing air" field with a description like "The air currents are very strong and very localized here...". Something to give the player some sense of what's going on there.
